src/clusterfuzz/_internal/crash_analysis/crash_analyzer.py

```diff
@@ -30,6 +30,21 @@
     'READ', 'UNKNOWN', 'UNKNOWN READ', 'UNKNOWN WRITE', 'WRITE',
     'Uncaught exception'
 ]
+CRASH_TYPES_NON_SECURITY = [
+  'Stack-overflow',
+  'Fatal-signal',
+  'Missing-library',
+  'Overwrites-const-input',
+  'ASSERT_NOT_REACHED', # Unexpected conditions reached in the program.
+  'Ill',
+  'Floating-point-exception',
+  'RUNTIME_ASSERT', # RUNTIME_ASSERT in V8 (not a crash, but is a sign of an error
+  'V8 correctness failure',
+  'Timeout',
+  'Out-of-memory',
+  'Unexpected-exit',
+  'Security CHECK failure', # Release SECURITY_CHECK in Blink shouldn't be marked as a security bug
+]
 SIGNAL_SIGNATURES_NOT_SECURITY = [
     'Sanitizer: ABRT',
```

Comment:
You probably still want to add the Illegal-signature one here. The reason why we have this separate check is there can be multiple stacktraces in the one output. We want to mark crashes in these cases as non security even if the crash type we end up getting is not one of these signal ones.

Reply:
by Illegal-signature you mean illegal-instruction, right? if so, added!
if not, where was the illegal-signature supposed to be in the stacktrace? i totally missed that

Reply:
Yep that was a typo, thanks!